mail_autosubscribe/models/mail_template.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2021 Camptocamp (http://www.camptocamp.com).
+# @author Iván Todorovich <ivan.todorovich@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class MailTemplate(models.Model):
+    _inherit = "mail.template"
+
+    use_autosubscribe_followers = fields.Boolean(default=True)
+
+    def _generate_template_recipients(
+        self, res_ids, render_fields, find_or_create_partners=False, render_results=None
+    ):
+        res = super()._generate_template_recipients(
+            res_ids, render_fields, find_or_create_partners, render_results
+        )
+
+        autosubscribe_followers = (
+            self.use_autosubscribe_followers
+            and not self.env.context.get("no_autosubscribe_followers")
+            # In this case, autosubscribers will be added by
+            # :func:`_message_get_default_recipients`
+            and not self.use_default_to
+            # and not self.env.context.get("tpl_force_default_to") does not exist more
+        )
+        if autosubscribe_followers:
+            for res_id in res.keys():
+                partners = (
+                    self.env["res.partner"].sudo().browse(res[res_id]["partner_ids"])
+                )
```

Comment:
Looks like now the "partner_ids" key is not always set by Odoo code. This leads to breaking tests for other modules in the this repository.
Maybe something like this, could help

```suggestion
               partner_ids = res[res_id].get("partner_ids")
               if not partner_ids:
                   continue
                partners = self.env["res.partner"].sudo().browse(partner_ids)
```

Reply:
@rlimaeco ping